client/src/components/common/applicant-info-nav.vue

```diff
@@ -2,7 +2,7 @@
   <v-col cols="5"
          class="text-right">
     <v-btn x-large
-           id="submit-back-btn"
+           :id="`submit-back-btn-${isValid}`"
```

Comment:
This breaks the selector in "main.sass" - `#submit-back-btn`.

Also line 14 below

Reply:
good catch.  forgot this completely.